Sources/AWSLambdaTesting/Lambda+Testing.swift

```diff
@@ -60,45 +64,25 @@ extension Lambda {
         }
     }
 
-    public static func test(_ closure: @escaping Lambda.StringClosure,
-                            with event: String,
-                            using config: TestConfig = .init()) throws -> String {
-        try Self.test(StringClosureWrapper(closure), with: event, using: config)
-    }
-
-    public static func test(_ closure: @escaping Lambda.StringVoidClosure,
-                            with event: String,
-                            using config: TestConfig = .init()) throws {
-        _ = try Self.test(StringVoidClosureWrapper(closure), with: event, using: config)
-    }
-
-    public static func test<In: Decodable, Out: Encodable>(
-        _ closure: @escaping Lambda.CodableClosure<In, Out>,
-        with event: In,
-        using config: TestConfig = .init()
-    ) throws -> Out {
-        try Self.test(CodableClosureWrapper(closure), with: event, using: config)
-    }
-
-    public static func test<In: Decodable>(
-        _ closure: @escaping Lambda.CodableVoidClosure<In>,
-        with event: In,
+    public static func test<Handler: LambdaHandler>(
```

Comment:
@tomerd Lambda.test only supports new async/await lambdas:
```swift
XCTAssertNoThrow(result = try Lambda.test(MyLambda.self, with: input))
```
The benefit is that the Lambda goes through the complete lifecycle with this approach. Initializer and handler are called.